iam_units/emissions.py

```diff
@@ -91,7 +100,6 @@
     "HFE365mcf3",
     "HFE374pc2",
     "HFE4310pccc124",
-    "HFE449sl",
```

Comment:
Maybe my previous comment https://github.com/IAMconsortium/units/pull/38/files#r896622533 was too hasty. Is this deliberate?